### ISSUE TITLE
Add proper labels to the `c-list-pagination` inputs

### ISSUE THREAD

**Issue by kelseymcmahon** [IntelliTect/Coalesce#651]
Currently when the DOM renders the inputs for the [c-list-page-size](https://coalesce.intellitect.com/stacks/vue/coalesce-vue-vuetify/components/c-list-page-size.html) and [c-list-page](https://coalesce.intellitect.com/stacks/vue/coalesce-vue-vuetify/components/c-list-page.html) the labels are not clear. For better accessibility practices we should create better labels (https://dequeuniversity.com/rules/axe/4.11/label-title-only?application=AxeChrome).

For the `c-list-page-size` the `aria-label` is currently "Open" and should be "Page Size".

For the `c-list-page` the `aria-label` does not exist. Instead of the "Current Page Number" used as the `title`, it should be set on the `aria-label`.
